src/error.rs

```diff
@@ -1,37 +1,37 @@
 use std::{
-    error, ffi,
+    error,
     fmt::{self, Debug, Display, Formatter},
 };
 
-use xenctrl_sys::{xc_error_code, xc_error_code_to_desc};
-
-#[derive(Copy, Clone)]
-pub struct Error(xc_error_code);
+pub struct Error {
```

Comment:
I think it is a little bit confusing that the error struct shadows the name of the general error interface.

Reply:
Right, we should renamed it `XcError`